content/news/2023-08-07-generic-tabular-plotter/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Generic scatter plots for any Galaxy tabular data"
+date: "2023-08-07"
+authors: Ross Lazarus
+authors_structured:
+- github: fubar2
+tease: "A tool generating interactive HTML scatter plots from Galaxy tabular data is now available for testing"
+hide_tease: false
+subsites: [all]
+---
+
+### Sample use case - Iris data scatterplot with mouse-over hover showing species in Galaxy
+
+Galaxy users can now easily turn this:
+
+![iris data](./iris_data_in_galaxy.png)
+
+into this interactive HTML output:
+
+<iframe src="https://lazarus.name/demo/Plotlytabular_Iris_data_on_iris_html.html" width="1024" height="768" resize="both" overflow="auto"  style="border:none;">
```

Comment:
do you want to point to the local file? Or remove the local file?

Reply:
Iframes will not display local html so removed the local copy - thanks!